### PR TITLE
`body` attribute required

### LINKED ISSUE #273: Is `body` required or optional?
I am not sure if the body is required and able to be blank (`{}`) or if it is optional. This isn't articulated in the spec. Trust ping protocol doesn't include it so that might mean it's optional but would like some clarification

### PR DESCRIPTION
**Pull request by TelegramSam**
Fixes #273

### CODE REVIEW

**Review by kdenhartog — APPROVED** [review]

**Review by awoie — APPROVED** [review]
lgtm

File: docs/spec-files/message_structure.md [hunk 1]
```diff
@@ -115,6 +115,8 @@ The `to` header cannot be used for routing, since it is encrypted at every inter
 
 - **expires_time** - OPTIONAL. Message Expired Time. The `expires_time` attribute is used for the sender to express when they consider the message to be expired, expressed in UTC Epoch Seconds (seconds since 1970-01-01T00:00:00Z UTC) [link](1970-01-01T00:00:00Z UTC). This attribute signals when the message is considered no longer valid by the sender. When omitted, the message is considered to have no expiration by the sender.
 
+- **body** - REQUIRED. The `body` attribute contains all the message type specific attributes of the message type indicated in the `type` attribute. This attribute must be present, even if empty.
```
Comment (awoie) [inline docs/spec-files/message_structure.md:118]:
```suggestion
- **body** - REQUIRED. The `body` attribute contains all the message type specific attributes of the message type indicated in the `type` attribute. This attribute MUST be present, even if empty.
```

Comment (awoie) [reply docs/spec-files/message_structure.md:118]: minor change, just making normative language uppercase.

**Comment by kdenhartog** [comment]
We tried to make a minor fix during the call today with suggestions and then merge this, but ended up causing a merge conflict. @TelegramSam once you get this one resolved, I think it's good to merge it per the discussions on the 6th of Septemeber WG call.

**Comment by TelegramSam** [comment]
I'm merging this given the above approval and trying to resolve repeated merge conflicts. :)

**Pull request merged**